### PR TITLE
Use wpi::Now for image capture timestamp

### PR DESCRIPTION
**Pull request by mcm001**
Uses wpi::Now() for camera -> process latency.

### CODE REVIEW

File: photon-core/build.gradle [hunk 1]
```diff
@@ -12,6 +12,13 @@ dependencies {
     implementation 'org.msgpack:msgpack-core:0.8.20'
     implementation 'org.msgpack:jackson-dataformat-msgpack:0.8.20'
 
+    // Wpiutil
+    compile "edu.wpi.first.wpiutil:wpiutil-jni:$wpilibVersion:linuxaarch64bionic"
+    compile "edu.wpi.first.wpiutil:wpiutil-jni:$wpilibVersion:linuxraspbian"
+    compile "edu.wpi.first.wpiutil:wpiutil-jni:$wpilibVersion:linuxx86-64"
+    compile "edu.wpi.first.wpiutil:wpiutil-jni:$wpilibVersion:osxx86-64"
```
Comment (Bankst) [inline photon-core/build.gradle:19]: Remove OSX
```suggestion
```

File: photon-core/src/main/java/org/photonvision/vision/pipeline/result/CVPipelineResult.java [hunk 2]
```diff
@@ -46,8 +47,8 @@ public CVPipelineResult(
     }
 
     public CVPipelineResult(
-            double processingMillis, double fps, List<TrackedTarget> targets, Frame outputFrame) {
-        this(processingMillis, fps, targets, outputFrame, null);
+            double processingNanos, double fps, List<TrackedTarget> targets, Frame outputFrame) {
```
Comment (Bankst) [inline photon-core/src/main/java/org/photonvision/vision/pipeline/result/CVPipelineResult.java:50]: If we're using nanoseconds, would a `long` not make more sense?

File: photon-core/src/main/java/org/photonvision/vision/pipeline/result/DriverModePipelineResult.java [hunk 3]
```diff
@@ -21,7 +21,7 @@
 import org.photonvision.vision.frame.Frame;
 
 public class DriverModePipelineResult extends CVPipelineResult {
-    public DriverModePipelineResult(double latencyMillis, double fps, Frame outputFrame) {
-        super(latencyMillis, fps, List.of(), outputFrame);
+    public DriverModePipelineResult(double latencyNanos, double fps, Frame outputFrame) {
```
Comment (Bankst) [inline photon-core/src/main/java/org/photonvision/vision/pipeline/result/DriverModePipelineResult.java:24]: Same here - `long` over `double`?

**Review by Bankst — CHANGES_REQUESTED** [review]

File: photon-core/src/main/java/org/photonvision/vision/pipeline/result/CVPipelineResult.java (hunk 2 above)
Comment (mcm001) [reply photon-core/src/main/java/org/photonvision/vision/pipeline/result/CVPipelineResult.java:50]: the MathUtils method I use to calculate this returns a double iirc

**Review by Bankst — DISMISSED** [review]

**Review by Bankst — DISMISSED** [review]

**Review by Bankst — APPROVED** [review]

**Pull request merged**